class/defaults.yml

```diff
@@ -38,6 +38,10 @@ parameters:
       replicaLabels:
         - prometheus_replica
         - rule_replica
+    query_alerts:
```

Comment:
Why don't we have `alerts` as a subkey of `query` etc.? We already need to handle `enabled` specially anyway, so I don't see a big issue having to handle `alerts` specially as well (unless `alerts` is a valid key to pass to the Jsonnet library, then we can't do that obviously).

Reply:
I tried to no "pollute" the params for the jsonnet library if they add parameters in the future.

Reply:
Fair enough, I think the current design is probably a mistake and should be something like the following instead:

```yaml
query:
  enabled: true
  alerts: { # whatever we need }
  params: { # values to pass to the lib }
```

Reply:
Yeah I'd like this design way more... But I'd like to not introduce a breaking change 😅

Reply:
Yeah, that comment wasn't meant that we should do this now, but I'll file an issue for it afterwards